decidim-core/lib/decidim/abilities.rb

```diff
@@ -3,5 +3,6 @@
 module Decidim
   module Abilities
     autoload :ParticipatoryProcessRoleAbility, "decidim/abilities/participatory_process_role_ability"
+    autoload :UserManagerAbility, "decidim/abilities/user_manager_ability"
```

Comment:
@deivid-rodriguez I am not sure why I need to add this because the tests were failing without it. I saw in one of your commits you removed the other lines 🤔